src/mockgyver.erl

```diff
@@ -1309,29 +1466,20 @@ group_fas_by_mod(MFAs) ->
                          MFAs),
     dict:to_list(ModFAs).
 
-get_exported_fas_and_object_code(Mod) ->
-    case get_exported_fas(Mod) of
-        {ok, FAs} ->
-            case code:get_object_code(Mod) of
-                {Mod, Bin, Filename} ->
-                    {ok, {FAs, Bin, Filename}};
-                error ->
-                    {error, {no_object_code, Mod}}
-            end;
-        {error, _}=Error ->
-            Error
-    end.
-
 get_exported_fas(Mod) ->
-    try
-        {ok, [{F, A} || {F, A} <- Mod:module_info(exports),
-                        {F, A} =/= {module_info, 0},
-                        {F, A} =/= {module_info, 1}]}
+    try Mod:module_info(exports) of
```

Comment:
Done. (My maybe overly defensive reason for the change phrasing of the code was that since the code catches `error:undef`, I wanted to make sure that any such `undef` could only ever come from `Mod:module_info(exports)` and not from any programming error inside `filter_fas/1`. But filter_fas is a local function which does not do any further calls, at least not currently.)

Reply:
Ah, I see